about.html

```diff
@@ -0,0 +1,135 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:wght@200;300;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="about.css" type="text/css">
+</head>
+
+<body>
+  <section class="header">
+    <div class="header-upper">
+      <ul>
+        <li><a href="#"><i class="fa fa-facebook" ></i></a></li>
+        <li><a href="#"><i class="fa fa-twitter" ></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+
+    <nav>
+      <div class="logo">
+        <a href="index.html"><img src="./images/new-logo.png" alt="Logo"></a>
+      </div>
+      <div class="menu-items">
+        <ul class="nav-link">
+          <li class="link"><a href="#" id="aboutit">About</a></li>
+          <li class="link"><a href="#">Program</a></li>
+          <li class="link"><a href="#">Join</a></li>
+          <li class="link"><a href="#">Sponsers</a></li>
+          <li class="link"><a href="#">News</a></li>
+
+        </ul>
+        <div class="header-button"><button type="button" class="button-header">G Gathering</button></div>
+      </div>
+    </nav>
+  </section>
+  <div id="mySidenav" class="sidenav">
+    <a href="#" class="closebtn" onclick="closeNav()">&times;</a>
+    <a href="about.html">About</a>
+    <a href="#">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsers</a>
+    <a href="#">News</a>
+  </div>
+  <section class="about" id="about">
+    <div class="main">
+      <div class="hamburger" onclick="openNav()"><img src="./images/Ham.jpg" alt="hamburger"></div>
+
+      <div class="content">
+        <h1>"Welcome! to the Marketing world"</h1>
+        <h2>A GLOBAL GATHERING <br>OF BRAND BUILDERS & MARKETERS</h2>
+        <div class="inner-content">
+          <p>The Marketing Conference spotlights the latest innovations and breakthroughs in the marketing and advertising industry. Connect with the global community and join thought leaders and storytellers as they explore visionary ideas that engage and
+            inspire
+          </p>
+        </div>
+        <div class="inner-content-bottom">
+          <p>Please contact as per Email for any further questions about global Gathering</p>
+          <h3>globalgather@ggather.org</h3>
+          <div class="under-line"> </div>
+
+        </div>
+      </div>
+
+    </div>
```

Comment:
Kindly try to match the `Behance` design for the header section in the mobile version on the about page. the `font size` need to be smaller, Please make the content exactly on the center, and Please change the `section` tag with the `header` tag for semantic purposes
| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157545029-9100df83-9f33-4378-ba0a-8f7bb2d24195.png) | ![screencapture-raw-githack-monika-soam-Marketing-Conference-About-Page-about-html-2022-03-10-00_04_53](https://user-images.githubusercontent.com/24703171/157544859-f9ccf0eb-cc0a-4bff-820f-54f01509d194.png) |

Reply:
This is fixed